test/watch.js

```diff
@@ -21,13 +21,15 @@ describe('gulp', function() {
 
     var writeTimeout = 125; // Wait for it to get to the filesystem
     var writeFileWait = function(name, content, cb) {
-      if (!cb) cb = function() {};
+      if (!cb) { cb = function() {} };
```

Comment:
should be on a new line, not sure if there is a JSCS rule for disallowing one-liners like this